core/src/main/res/drawable/ic_notifications_white.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M12,22c1.1,0 2,-0.9 2,-2h-4c0,1.1 0.89,2 2,2zM18,16v-5c0,-3.07 -1.64,-5.64 -4.5,-6.32L13.5,4c0,-0.83 -0.67,-1.5 -1.5,-1.5s-1.5,0.67 -1.5,1.5v0.68C7.63,5.36 6,7.92 6,11v5l-2,2v1h16v-1l-2,-2z"/>
```

Comment:
Please set `android:fillColor` to `#FFFFFFFF` instead of using `android:tint`. I know that Android Studio does that but I think using the colour directly is more clean :)

Reply:
@ByteHamster 
for all the remaining icons used it is filled with "#FF000000" so i think its fine to use it for this icon as well!!